### ISSUE TITLE
create-version-tag passes if version does not exist

### ISSUE THREAD

**Issue by tinexw** [pact-foundation/pact_broker-client#41]
When the `create-version-tag` command is called with a non-existing version, it passes. 

This can lead to the case that a latest tag is not shown in the UI anymore.

1. Tag an existing version with `prod` -> `prod` tag is shown
```
./pact-broker create-version-tag -a messaging-app -b http://localhost -e feaba6227a48e03c0eeb1c4a27d685496349e78d -t prod
Tagging messaging-app version feaba6227a48e03c0eeb1c4a27d685496349e78d as prod
```

![image](https://user-images.githubusercontent.com/7386945/46873818-dcef6600-ce37-11e8-9a84-685b9865c058.png)

2. Tag a non-existing version with `prod` -> no latest `prod` tag is shown
```
./pact-broker create-version-tag -a messaging-app -b http://localhost -e 123 -t prod
Tagging messaging-app version 123 as prod
```
![image](https://user-images.githubusercontent.com/7386945/46874069-90f0f100-ce38-11e8-81be-f6fb3f0b22cc.png)

![image](https://user-images.githubusercontent.com/7386945/46874019-73bc2280-ce38-11e8-8430-ebf0d2401601.png)

I would prefer if the command would fail if the version does not exist.

**Comment by bethesque** [comment]
I can add a flag to make it fail if the version does not exist. What's happening is that it's creating the version for you when you create the tag (which is behaviour required by the original tagging code). In the meantime, you could add a describe-version check beforehand to make sure the version exists before you tag it.

**Comment by tinexw** [comment]
That flag would be great! We usually create the versions by running JUnit tests and publishing the results. Thus, we don't want to create additional versions apart from that. It happened accidentally in the past and was quite confusing.

**Comment by anto-ac** [comment]
@bethesque Did you ever get round to adding the flat? We were hit by this today and it took a bit of time to figure out what was going on.

**Comment by bethesque** [comment]
I've released a new version of the client, pact-cli docker image and pact ruby standalone with the fix. It's a bit naughty, but I've changed the default behaviour to raise an error if the version doesn't exist (not backwards compatible, but I think, for the best. We'll consider it a bug fix, not a feature change!). If anyone wants the version to be auto created, then there's the --auto-create-version flag.
